pkg/sloop/processing/relationship.go

```diff
@@ -0,0 +1,126 @@
+package processing
+
+import (
+	"encoding/json"
+	"github.com/dgraph-io/badger/v2"
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/pkg/errors"
+	"github.com/salesforce/sloop/pkg/sloop/kubeextractor"
+	"github.com/salesforce/sloop/pkg/sloop/store/typed"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped/badgerwrap"
+)
+
+
+
+
+//  transfer watchRec.Payload to Relationship payload: Convert watchRec to record in relationship table and save key-value pair
+func updateRelationshipTable(tables typed.Tables, txn badgerwrap.Txn, watchRec *typed.KubeWatchResult, metadata *kubeextractor.KubeMetadata) error {
+	if watchRec.Kind != kubeextractor.PodKind {
+		glog.Infof("Skipping relationship table update as kubewatch result is not PodKind")
+		return nil
+	}
+	// convert node -> pod relationship from watchRec
+	// get new relationship record from watch record
```

Comment:
are these comments needed? do they add anything more then the name of the method being called? If not, please remove them - they are a maintenance problem. If there is some extra value, can we encapsulate that value in the method name instead?

Reply:
please apply this to any comments - the comments should add value to the reader - not just repeat what the code is telling us. The code should tell us what is being done, the comments tell us why or the abstraction behind it.

Reply:
Prefer to keep those comments for explanation purpose, right now the table only support node->pod mapping